src/app/components/Input/Input.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from './Input.module.css';
+
+export type InputProps = {
+  label: string;
+  id: string;
+  value: string;
+  type: 'email' | 'password';
+  placeholder: string;
+  onChange: () => void;
+};
+
+const Input = ({
+  label,
+  id,
+  value,
+  type,
+  placeholder,
+  onChange,
+}: InputProps): JSX.Element => {
+  return (
+    <div>
+      <label htmlFor={id} className={styles.label}>
+        {label}
+      </label>
+      <input
+        className={styles.input}
+        type={type}
+        value={value}
+        placeholder={placeholder}
+        onChange={onChange}
```

Comment:
What happens here onChange? I guess you want to receive the value that is currently in the input field. 
```suggestion
        onChange={event => onChange(event.target.value)}
```